whole-tale/ngx-dashboard
# Disable create-tale-modal without name/env

## Problem
The "Create" button on the `create-tale-modal` is enabled even when the input is invalid. This will likely throw an error that could easily be prevented by disabling the button when necessary.
  
## Approach
Disable the "Create" button if no name has been entered or if no environment has been chosen.

NOTE: This also contains a minor CSS change to add a margin below the search filter

## How to Test
1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Navigate to the Tale Catalog view
4. Click the "Create New Tale" button
    * You should see the Create Tale modal open
    * You should see the "Create" button at the bottom is disabled at first
5. Enter a name and choose an environment
    * You should see that the "Create" button is enabled only after filling both inputs
